contrib/pg_tde/documentation/CONTRIBUTING.md

```diff
@@ -1,10 +1,90 @@
+# Contributing guide
```

Comment:
Shouldn't we put this file (also) to a more visible place, like the top level folder in the git repository?

Reply:
I guess we can. But this one relates to pg_tde specifically. If we place it on top of the repo, do we need to make it more universal?

Reply:
I'm for either option:)